packages/sources/tiingo/README.md

```diff
@@ -230,6 +230,8 @@ Aliases: `fx`, `commodities`
 
 https://api.tiingo.com/documentation/forex
 
+This endpoint has the ability to leverage inverses in the scenario a specific pair exists but not its inverse on the Tiingo forex API.
```

Comment:
This will get lost because the READMEs are auto-generated from code.

Add it to `packages/sources/tiingo/src/endpoint/forex.ts`'s `description`.

Reply:
In src/endpoint/forex.ts:
```
export const description = "..."
```

Reply:
I've updated this in the latest commit. But I noticed afterwards Tiingo is in the readmeBlacklist.json.

Reply:
Hmm I wonder why it's still on the blacklist. Can you try removing it and running `yarn generate:readme tiingo` and see if it overwrites the README properly?

Reply:
The generated README looks good to me

Reply:
The latest commit should take care of this